operators/admin/olm-cs-podsched.adoc

```diff
@@ -6,26 +6,39 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-When an Operator Lifecycle Manager (OLM) catalog source of source type `grpc` defines a `spec.image`, the Catalog Operator creates a pod that serves the defined image content. By default, this pod defines the following in its spec:
+When an Operator Lifecycle Manager (OLM) catalog source of source type `grpc` defines a `spec.image`, the Catalog Operator creates a pod that serves the defined image content. By default, this pod defines the following in its specification:
 
 * Only the `kubernetes.io/os=linux` node selector
 * No priority class name
 * No tolerations
 
 As an administrator, you can override these values by modifying fields in the `CatalogSource` object's optional `spec.grpcPodConfig` section.
 
+[IMPORTANT]
+====
+The Marketplace Operator, `openshift-marketplace`, manages the default `OperatorHub` custom resource's (CR) `CatalogSource` objects. If you attempt to modify fields in the `CatalogSource` object’s `spec.grpcPodConfig` section, the Marketplace Operator automatically reverts these modifications. This behavior occurs because the Marketplace Operator manages each default `CatalogSource` object at a global level, so that all users in a cluster's namespace must receive the same default object configuration.
```

Comment:
Hi @kevinrizza . My proposed updated note changes the original meaning. Is this correct? 

The Marketplace Operator does not revert any changes to `spec.grpcPodConfig section`, so maybe this behaviour applies to local catlog sources and not the global catalog sources?

Reply:
That honestly sounds like a bug rather than expected behavior. The marketplace operator owns those catalogs, so we shouldn't document any expected behavior around being able to modify them without disabling the marketplace's ownership over those resources

Reply:
Hi @jianzhangbjz . The Marketplace Operator should have complete control over the state resources until a user explicitly disables the operator. Have you witnessed different behaviour? If so, this behaviour would indicate a bug exists with the operator?

Reply:
yes, I met it when I reviewed this doc, I'm double checking it.

Reply:
> Version(s):
4.14 through to 4.10

Hi @dfitzmau and @kevinrizza , it can be reverted on OCP 4.13 and 4.14, but not 4.12, 4.11, 4.10, reported a bug here: https://issues.redhat.com/browse/OCPBUGS-15384